kubernetes/assets/dashboards/kubernetes_pods.json

```
@@ -0,0 +1,1049 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Kubernetes - Pods",
```

```suggestion
    "board_title": "Kubernetes Pods Overview",
```
`-` in the title will make the url look like `/kubernetes---pod`.